tests/apple-exporter/components/test-class-pre.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+require_once __DIR__ . '/class-component-testcase.php';
+
+use Apple_Exporter\Components\Pre as Pre;
+
+class Pre_Test extends Component_TestCase {
+
+  public function testBuildingRemovesTags() {
+    $component = new Quote( '<pre><code data-lang="javascript">var x = {
```

Comment:
Looks like you're calling the wrong component here, which is leading to a unit test failure.